### DISCUSSION TITLE
Why chartjs-plugin-zoom does not work for chartjs-chart-treemap library?

### DISCUSSION THREAD

**Discussion by mirzaeianfar** [chartjs/chartjs-plugin-zoom#603] [Q&A]
How can i use chartjs-plugin-zoom for chartjs-chart-treemap library?
When i use it in chartjs-chart-treemap library,it does not work and i can not zoom.
Please help me.

**Comment by LeeLenaleee** [comment]
I assume it is because the treemap does not use any scales and from a quick look into the zoom source code it relies on scales in some parts.

**Reply by mirzaeianfar** [reply]
Can you please tell me how I can zoom in on the treemap.
I desperately need to be able to zoom in on the treemap.

**Reply by kurkle** [reply]
There is nothing the zoom plugin can do about the fact that treemap is not using scales and thus can not be scaled.
